src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -1699,6 +1699,26 @@ export default class Aircraft {
     }
 
     /**
+     * Checks if the aircraft is inside the airspace of a specified airport
+     *
+     * @for AircraftInstanceModel
+     * @method isInsideAirspace
+     * @param  {airport} airport the airport whose airspace we are checking
+     * @return {Boolean}
+     * @private
+     */
+    isInsideAirspace(airport) {
+        let withinAirspaceLateralBoundaries = this.distance <= airport.ctr_radius;
+        const withinAirspaceAltitudeRange = this.altitude <= airport.ctr_ceiling;
+
+        if (_has(airport, 'perimeter')) {    // polygonal airspace boundary
+            withinAirspaceLateralBoundaries = point_in_area(this.position, airport.perimeter);
+        }
+
+        return withinAirspaceAltitudeRange && withinAirspaceLateralBoundaries;
```

Comment:
yes!!! 👍 

one thing to remember and I don't think we've gone over this yet, so I'll let this go as is this time.  When a method returns a boolean or a constant is a boolean, it should be prepended with `is`.  so instead of 
```javascript
return isWithinAirspaceAltitudeRange && isWithinAirspaceLateralBoundaries;
```
this reads a little better and you can immediately tell that those two things are booleans.